ui/imports/shared/popups/send/views/AmountToSendNew.qml

```diff
@@ -98,7 +98,7 @@ Control {
 
         readonly property string inputDelocalized:
             root.valid && textField.length !== 0
-                ? textField.text.replace(",", ".") : "0"
+                ? textField.text.replace(LocaleUtils.userInputLocale.decimalPoint, ".") : "0"
```

Comment:
Here please use `root.decimalPoint`. Taking it from outside makes this component easier to test because there is no global state potentially affecting testing environment.

Reply:
Fixed